service/controller/key/key.go

```diff
@@ -0,0 +1,18 @@
+package key
```

Comment:
What is the idea with this key package? I do not quite follow.

Reply:
When the operator is flattened we'll have the `key` package here. See https://github.com/giantswarm/release-operator/tree/master/service/controller/key